packages/components/psammead-useful-links/src/index.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import styled from 'styled-components';
+import { getSerifMedium } from '@bbc/psammead-styles/font-styles';
+import { C_EBON } from '@bbc/psammead-styles/colours';
+import { grid } from '@bbc/psammead-styles/detection';
+import { getPica } from '@bbc/gel-foundations/typography';
+import { string, arrayOf, shape } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import {
+  GEL_SPACING, // 8 px
+} from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+
+const UsefulLinkWrapper = styled.div`
+  padding-top: ${GEL_SPACING};
+`;
+
+export const UsefulLinkItem = styled.a`
+  ${({ script }) => script && getPica(script)};
+  ${({ service }) => getSerifMedium(service)};
+  color: ${C_EBON};
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+    cursor: pointer;
+  }
+`;
+
+export const UsefulLinksUl = styled.ul`
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    @supports (${grid}) {
+      display: grid;
+      grid-template-columns: auto auto;
+    }
+  }
+`;
```

Comment:
To avoid passing the role="list" prop in the stories or Simorgh container, can you do something like:

```
const StyledUsefulLinksUl =  styled.ul`
  ...
`

export const UsefulLinksUl = ({ children, ...props }) => (
  <StyledUsefulLinksUl role="list" {...props}>
    {children}
  </StyledUsefulLinksUl>
);
```